samples/40.timex-resolution/src/main/java/com/microsoft/bot/sample/timex/resolution/LanguageGeneration.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.timex.resolution;
+
+import com.microsoft.recognizers.datatypes.timex.expression.TimexProperty;
+
+import java.time.LocalDateTime;
+
+/**
+ * This language generation capabilities are the logical opposite of what the recognizer does.
+ * As an experiment try feeding the result of language generation back into a recognizer.
+ * You should get back the same TIMEX expression in the result.
+ */
+public class LanguageGeneration {
+    private static void describe(TimexProperty t) {
+        // Note natural language is often relative, for example the sentence "Yesterday all my troubles seemed so far away."
+        // Having your bot say something like "next Wednesday" in a response can make it sound more natural.
+        LocalDateTime referenceDate = LocalDateTime.now();
+        System.out.println(t.getTimexValue() + t.toNaturalLanguage(referenceDate));
```

Comment:
```suggestion
        String output = String.format("%s %s", t.getTimexValue(), t.toNaturalLanguage(referenceDate));
        System.out.println(output);
```

[C#](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/40.timex-resolution/LanguageGeneration.cs#L21) has a space between both values